Userland/Applications/FileManager/main.cpp

```diff
@@ -520,7 +520,7 @@ ErrorOr<int> run_in_desktop_mode()
     return GUI::Application::the()->exec();
 }
 
-ErrorOr<int> run_in_windowed_mode(String initial_location, String entry_focused_on_init)
+ErrorOr<int> run_in_windowed_mode(const String& initial_location, const String& entry_focused_on_init)
```

Comment:
pls use east const kthx

Reply:
Can you put the fix in the first commit please? You fixed it in the second and now the diff is unnecessarily complicated.